pkg/search/postgres/common.go

```diff
@@ -325,8 +326,56 @@ func (p *parsedPaginationQuery) AsSQL() string {
 	return paginationSB.String()
 }
 
-func standardizeQueryAndPopulatePath(q *v1.Query, schema *walker.Schema, queryType QueryType) (*query, error) {
+func enrichQueryWithSACFilter(ctx context.Context, q *v1.Query, schema *walker.Schema, queryType QueryType) (*v1.Query, error) {
+	switch queryType {
+	case DELETE:
+		if schema.PermissionChecker != nil {
+			if ok, err := schema.PermissionChecker.WriteAllowed(ctx); err != nil {
+				return nil, err
+			} else if !ok {
+				return nil, sac.ErrResourceAccessDenied
+			}
+			return q, nil
+		}
+		sacFilter, err := GetReadWriteSACQuery(ctx, schema.ScopingResource)
+		if err != nil {
+			return nil, err
+		}
+		pagination := q.GetPagination()
+		query := searchPkg.ConjunctionQuery(sacFilter, q)
+		query.Pagination = pagination
+		return query, nil
+		/*
+			case GET:
+				if schema.PermissionChecker != nil {
+					if ok, err := schema.PermissionChecker.ReadAllowed(ctx); err != nil {
+						return nil, err
+					} else if !ok {
+						return getMatchNoneQuery(), nil
+					}
+					return q, nil
+				} else {
+					sacFilter, err := GetReadSACQuery(ctx, schema.ScopingResource)
+					if err != nil {
+						return nil, err
+					}
+					pagination := q.GetPagination()
+					query := searchPkg.ConjunctionQuery(sacFilter, q)
+					query.Pagination = pagination
+					return query, nil
+				}
+
+		*/
```

Comment:
Could you delete it or add a comment with TODO and explanation why it's commented and when it could be uncomented?

Reply:
Why tests passes with this branch commented out?